models/data.js

```diff
@@ -21,21 +21,18 @@
 var _ = require("underscore");
 var fs = require("fs");
 
-function reader_norm(out, path, callback) {
+function reader_norm(out, pathname, callback) {
   // TODO: more reliable checking if file is data file
-  var files = fs.readdirSync(path);
-
-  console.log("files: ", files);
-
+  var files = fs.readdirSync(pathname);
   _.each(files, function (file) {
```

Comment:
is `files` an array? if so, can we just use `forEach`? i think the benefit of using `_.each` is that it works on objects too, but i see it used pretty liberally in the codebase, can you do a quick audit in the file and replace with `forEach` where appropriate?